components/VideoPlayer.vue

```diff
@@ -48,7 +51,12 @@ export default Vue.extend({
         try {
           await this.player.load(newSource);
         } catch (e) {
+          // eslint-disable-next-line no-console
           console.error('Error code', e.code, 'object', e);
```

Comment:
I think this should also be made into an error modal. Perhaps with something like:

```
Playback failed

Player error code: XYZ
Message: ...
```

Just make it more explicit.